Makefile

```diff
@@ -79,7 +79,8 @@ install: .bin/packr2
 .PHONY: docker
 docker: .bin/packr2
 		packr2 || (GO111MODULE=on go install github.com/gobuffalo/packr/v2/packr2 && packr2)
-		CGO_ENABLED=0 GO111MODULE=on GOOS=linux GOARCH=amd64 go build
+		CGO_ENABLED=0 GO111MODULE=on GOOS=linux GOARCH=amd64 go build \
```

Comment:
Hope you don't mind — it adds a version other than `master` to logs when you run it locally.

Reply:
👍